src/http/conn.rs

```diff
@@ -235,12 +237,17 @@ where I: AsyncRead + AsyncWrite,
 
         let wants_keep_alive = head.should_keep_alive();
         self.state.keep_alive &= wants_keep_alive;
-        let mut buf = Vec::new();
+
+        // TODO: Can we reuse this butter?
+        // TODO: Figure out a better init capacity
+        let mut buf = BytesMut::with_capacity(0);
```

Comment:
The only reason I was using `BytesMut` is because of the Tokio docs and types. I'll change them back to `Vec`.